### DISCUSSION TITLE
v1.8.8 - Sequoia Compatibility & Bug Fixes

### DISCUSSION THREAD

**Discussion by nodeful** [bitgapp/eqMac#936] [General]
* Fixed AirPods going into switching loop for some users
* Fixed issue that prevented eqMac from working on macOS Sequoia
* Fixed Expert Preset Import not working
* Fixed QuickLook sound not being routed when App Mixer is enabled
* Added ability to choose how to route System Alert sounds
* Other minor bug fixes and improvements

<hr /><em>This discussion was created from the release <a href='https://github.com/bitgapp/eqMac/releases/tag/v1.8.8'>v1.8.8 - Sequoia Compatibility & Bug Fixes</a>.</em>

**Comment by mxmkmarquette** [comment]
is that normal that its not working properly ? like i got sequoia beta 3, and when activating it it either dont play or bug

**Reply by nodeful** [reply]
Just updated to Sequoia Beta 3 and eqMac still working as intended. Could you please provide more information as to how it's failing? Just no sound? Which app is producing audio as well as what audio device are you using and at what sample rate.

**Comment by yarecco** [comment]
-Since Sequoia public Beta EqMac does not stay in login item list. No matter what you do it won't start at login. Need to be started manually every time.

**Comment by angusgastle** [comment]
Whatever you did on the last update fixed the app perfectly. Of course, I've moved to Sequoia Public Beta and the issue has re-appeared. This is just for visibility that the audio (AirPods going into switching loop) is going to come back in the next edition. Still hands down the best software for audio. Gamechanger.
